src/lug/Window/Window.cpp

```diff
@@ -202,6 +202,9 @@ bool Window::pollEvent(Event& event) {
             close();
             return false;
         }
+        else if (!value) {
```

Comment:
Why didn't you check directly the call to `_impl->pollEvent` and returning false directly?